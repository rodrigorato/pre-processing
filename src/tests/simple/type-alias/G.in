class Foo { 
	alias fooMain = public static main(String[] args);

	alias dbl = Double;

	public static dbl m (int s) {
		dbl a = 2.0, b = 1.99;
		dbl c;

		return (dbl) a + b;
	}

	alias castToDouble = (dbl);
	alias itg = Integer;
	alias zero = 0;

	public static dbl m (itg s) {
		return castToDouble(itg.valueOf(2.0) + zero);
	}

	fooMain {
		System.out.println("This test has 24 lines :O");
	}
}
