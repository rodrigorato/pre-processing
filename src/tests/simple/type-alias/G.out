class Foo { 
	

	

	public static Double m (int s) {
		Double a = 2.0, b = 1.99;
		Double c;

		return (Double) a + b;
	}

	
	
	

	public static Double m (Integer s) {
		return (Double)(Integer.valueOf(2.0) + 0);
	}

	public static main(String[] args) {
		System.out.println("This test has 24 lines :O");
	}
}
